operator/controllers/seldondeployment_prepackaged_servers.go

```diff
@@ -284,6 +284,11 @@ func createStandaloneModelServers(r *SeldonDeploymentReconciler, mlDep *machinel
 			deploy = createDeploymentWithoutEngine(depName, seldonId, sPodSpec, p, mlDep, podSecurityContext)
 		}
 
+		// apply serviceAccountName defined in predictors to prepackaged containers
```

Comment:
Could we change this to `apply serviceAccountName to pod to enable EKS fine-grained IAM roles`? Just thinking it would be good to have a reminder later on why this was done.